ruby-challenges.rb

```diff
@@ -10,7 +10,22 @@
 reposts3 = 221
 # Expected output: '221 is odd'
 
+def even_or_odd(number)
+    if number % 2 == 0
+        p "#{number} is even"
+    elsif number % 2 != 0
+        p "#{number} is odd"
+    end
+end
+
+even_or_odd(reposts1)
+even_or_odd(reposts2)
+even_or_odd(reposts3)
+
```

Comment:
Well done, but don't forget Ruby's built-in methods!  (ie even? odd? means cleaner code)